signbank/dictionary/csv_import.py

```diff
@@ -816,6 +816,8 @@ def import_manual_validation(request):
         for row in validation_record_reader:
             if validation_record_reader.line_num == 1:
                 continue
+            if ":" not in row["idgloss"]:
```

Comment:
issue: the client has said this isn't needed 🙂

Reply:
Oh, SNAP 😆